docs/source/user_guide/tutorials.rst

```diff
@@ -39,32 +39,50 @@ Get the notebooks
 If you have the *landlab* source code, you already have the notebooks (they are
 in the *notebooks/* folder).
 
-If you don't have a copy of the source code, you can run the
-`notebooks.py`_
+If you don't have a copy of the source code, you can run the `notebooks.py`_
 script to fetch the set of notebooks that matches your version of *landlab*.
 This can be done either by running,
 
 .. code-block:: bash
 
-   $ curl -L https://raw.githubusercontent.com/landlab/landlab/master/notebooks.py | python -
+   curl -L https://raw.githubusercontent.com/landlab/landlab/master/notebooks.py | python -
```

Comment:
I think comments should be added for this line, it's unclear how we can select the version which is spoken about L. 56.

Reply:
I agree. I'll add text that describes how this works.

Reply:
@sebastien-lenard I've added an example that downloads a specific version of the notebooks.